test/syscalls/linux/msgqueue.cc

```diff
@@ -654,6 +659,176 @@ TEST(MsgqueueTest, MsgOpGeneral) {
   }
 }
 
+// Test msgctl with IPC_STAT option.
+TEST(MsgqueueTest, MsgCtlIpcStat) {
+  const time_t start = time(nullptr);
```

Comment:
For consistency with other tests, could we use `absl::Now`? For example: https://cs.opensource.google/gvisor/gvisor/+/master:test/syscalls/linux/wait.cc;l=424?q=absl::Now&ss=gvisor%2Fgvisor

Ditto throughout.

Reply:
Done.

One interesting thing is, the time values in the queue are of type `time_t`, so to compare them with `Time` we can either do `absl::FromTimeT(time_t_var)` or `absl::ToTimeT(start)`. The tests only work if we use `ToTimeT`, `FromTimeT` fails with a time diff of 100ms to 200ms. They also pass if we don't use `Time` struct at all. My guess is that `Time` has a higher sensitivity than `time_t` and `FromTimeT` rounds `time_t` down, so there's always this small diff.